### PR TITLE
Bugfix/lattice 2364 hazelcast queue config

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java [hunk 1]
```diff
@@ -21,6 +22,7 @@
     private final        List<Class<?>>                     additionalPods = new ArrayList<>();
     public static final  Class<?>[]                         DEFAULT_PODS   = new Class<?>[] {
             AsyncPod.class,
+            Auth0Pod.class,
```
Comment (makosblade) [inline src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: What is this added for?

Comment (makosblade) [reply src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: /I think we want to leave it up to extending classes to decide whether to use `Auth0Pod` or not

Comment (you-leee) [reply src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: All of them are registering Auth0Pod, where they are using the default pods, so I figured we could add it here as well. I forgot to delete it from `NeuronServer` though

Comment (makosblade) [reply src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: I think we should take the `Auth0Pod` out of here b/c Rhizome is meant to be as small/generic as possible and is technically independent from the OpenLattice stack

Comment (makosblade) [reply src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: We could actually add another class that extends `RhizomeApplicationServer` that includes `Auth0Pod`, `JdbcPod` and all of the other pods that are always present across `Datastore`  and `Conductor`

Comment (you-leee) [reply src/main/java/com/kryptnostic/rhizome/core/RhizomeApplicationServer.java]: Okay, that seems like a better idea!

**Pull request merged**